### ISSUE TITLE
English update

### ISSUE THREAD

**Issue by jeroenhabets** [NewLifeX/NewLife.Redis#107]
Hi,

We've been happy and grateful users of this library. Thanks @Soar360  for translating your readme back then [#84](https://github.com/NewLifeX/NewLife.Redis/issues/84) though I noticed it's not been updated since.
NOTE: On NuGet clicking the "[ENGLISH]" link does not work (I think it is supposed to go to [Readme.en.md](https://github.com/NewLifeX/NewLife.Redis/blob/master/Readme.en.md)?) 

We're now contemplating updating our Redis from 6.2 to 7.0 and I tried to find out which Redis versions you support. I.e. from which version onward is 7.0 supported? And does upgrading your library have any breaking changes?

So to summarize:
- Could you fix the [ENGLISH] link on your NuGet readme?
- Could you update the English Readme.en.md (and ideally keep it up-to-date 😄)
- Could you add a table/info of Redis versions you support? (and for which versions of your package)
- Could you add release notes ([CHANGELOG.md](https://keepachangelog.com)), a.o. containing any breaking changes, we'd need to be aware of? 

As this may be a bit much to ask...if not, we'll try to figure it out ourselves. Thanks!

**Comment by Soar360** [comment]
https://github.com/NewLifeX/NewLife.Redis/pull/143
